.gitignore

```diff
@@ -8,3 +8,6 @@ node_modules
 telemetryCache
 quickStart.html
 .gitcommit
+
+# Auto generated Service Clients
+src/shared/telemetry/clienttelemetry.d.ts
```

Comment:
Should we move generated clients to something like `src/shared/generated-clients/telemetry/telemetry.d.ts`?

Reply:
Good idea - I'll put that in the backlog